doc/whats_new.rst

```diff
@@ -164,6 +167,9 @@ API
 
     - An ``overwrite=False`` default parameter has been added to :func:`write_source_spaces` to protect against accidental overwrites, by `Eric Larson`_
 
+    - An ``egi`` argument has been added to :func:`mne.channels.read_dig_montage` to allow reading in EGI MFF digitization coordinate files. The argument takes the path to the coordinates xml file as a string or defaults to None. by `Matt Boggess`_
```

Comment:
Technically you don't need this bit. We typically put changes there that could break people's code. Yours is more of an enhancement so the above text suffices. But it's okay to leave it